### PR TITLE
fix(encode): maximum call stack size exceeded for large compiled cair…

### PR DESCRIPTION
**Pull request by FawadHa1der**
maximum call stack size exceeded error when the cairo file size is larger than 8 mb. I have put in the fix.

### CODE REVIEW

File: src/utils/encode.ts [hunk 1]
```diff
@@ -4,7 +4,9 @@ export const IS_BROWSER = typeof window !== 'undefined';
 const STRING_ZERO = '0';
 
 export function arrayBufferToString(array: ArrayBuffer): string {
-  return String.fromCharCode.apply(null, array as unknown as number[]);
+  return new Uint8Array(array).reduce(function (data, byte) {
```
Comment (janek26) [inline src/utils/encode.ts:7]: can we have an arrow function here?

**Review by janek26 — COMMENTED** [review]
Thanks for your contribution and for discovering the bug! One minor comment

File: src/utils/encode.ts (hunk 1 above)
Comment (FawadHa1der) [reply src/utils/encode.ts]: done

**Comment by janek26** [comment]
ah, please base off of develop branch and also contribute back to develop please!

**Pull request closed without merging**